README.md

```
@@ -426,3 +445,7 @@ See the full release history and changes in the [CHANGELOG.md](CHANGELOG.md) fil
 ## Contributing
 
 We welcome contributions! Please read the [CONTRIBUTING.md](CONTRIBUTING.md) guide before submitting issues or pull requests.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)